Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/OwnershipSecurity.ts

```diff
@@ -22,8 +22,8 @@ export async function rejectByOwnershipSecurity(
   const functionName = 'OwnershipSecurity.rejectByOwnershipSecurity';
   Logger.info(functionName, frontendRequest.traceId, frontendRequest);
 
+  // RND-234 If it's a GET request and a descriptor, ignore ownership
   if (
-    // RND-234 If it's a GET request and a descriptor, ignore ownership
```

Comment:
Can you remove `RND-234` from this? It really doesn't have anything to do with that. Not sure why someone wrote that.

Reply:
By the way, I was accidentally looking at only the most recent commit when I wrote that. So I thought you had just moved an existing comment from before you started. Oops.

Anyway, RND-234 was the wrong one and doesn't really need to be mentioned.